nixos/modules/profiles/all-hardware.nix

```diff
@@ -104,6 +104,7 @@ in
       "mp8859"
 
       # USB drivers
+      "reset-raspberrypi"
```

Comment:
(Assertion unverified at this point in time)

This will break the stage-1 build due to a missing kernel module for older kernel versions, or kernels not configured with 
`CONFIG_RESET_RASPBERRYPI=m`.

Furthermore I'm not sure it should be with "USB drivers" considering it's not strictly the USB driver, it's closer to *power supply drivers* and such things. These are all somewhat messy to name and define :/.